### DISCUSSION TITLE
Like, Dislike and other short buttons

### DISCUSSION THREAD

**Discussion by Rickyrvxlovet** [anddea/revanced-patches#449] [Q&A] [answered]
How can I change these to normal? 

![Screenshot_2024-05-11-09-49-32-374_com google android youtube](https://github.com/anddea/revanced-patches/assets/138698343/10b944e0-1566-442b-9fa4-f14122609825)

**Comment by anddea** [comment, marked as answer]
Exclude `Shorts overlay buttons` patch. Or change to any other icon type in the same patch.
